examples/getting-started-movielens/03-Training-with-TF.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>We can move the export code to the 4th notebook?</p><p> </p><p>We can save the TensorFlow model.</p><p> </p><p>In the 4th notebook, we can load the NVTabular workflow and TensorFlow model to create the graph with Merlin Systems</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>